jsapp/js/components/RESTServices/RESTServicesForm.tsx

```diff
@@ -429,7 +476,7 @@ export default class RESTServicesForm extends React.Component {
                 type='text'
                 placeholder={t('Service Name')}
                 value={this.state.name}
-                errors={this.state.nameError}
+                errors={this.state.nameError || undefined}
```

Comment:
this looks weird, let's just set `nameError` to `undefined` instead of `null` in the first place?